### DISCUSSION TITLE
Generate platform specific wheel

### DISCUSSION THREAD

**Discussion by thewh1teagle** [takluyver/flit#708] [Q&A]
How can I generate platform specific wheel? I added `data` folder with pre compiled library but the wheel is still generated as platform independent 
https://github.com/thewh1teagle/aec-rs/blob/main/pyaec/pyproject.toml

**Comment by takluyver** [comment]
That's not a thing Flit can do, sorry: it's only for platform independent packages. If you want to make different packages for different platforms, you'll need to use a different build system.
